### PR TITLE
Document four unsupported mysql syntax and statements

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs/issues/7226

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by easonn7** [comment]
LGTM

File: mysql-compatibility.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ However, some features of MySQL are not supported. This could be because there i
 + `CHECK TABLE` syntax [#4673](https://github.com/pingcap/tidb/issues/4673)
 + `CHECKSUM TABLE` syntax [#1895](https://github.com/pingcap/tidb/issues/1895)
 + `GET_LOCK` and `RELEASE_LOCK` functions [#14994](https://github.com/pingcap/tidb/issues/14994)
++ `HANDLER` commands
```
Comment (morgo) [inline mysql-compatibility.md:42]: Can we refer to it as [`HANDLER` statement](https://dev.mysql.com/doc/refman/8.0/en/handler.html)?

We have some other statements not supported if you can document at the same time:
https://github.com/pingcap/docs/issues/7981
https://github.com/pingcap/docs/issues/7980

Comment (shichun-0415) [reply mysql-compatibility.md:42]: > Can we refer to it as [`HANDLER` statement](https://dev.mysql.com/doc/refman/8.0/en/handler.html)?
> 
> We have some other statements not supported if you can document at the same time: #7981 #7980

OK. I will document them as well. But do you know whether they have any associated issues? It seems other unsupported features are linked to issues.

**Review by morgo — APPROVED** [review]

File: mysql-compatibility.md (hunk 1 above)
Comment (morgo) [reply mysql-compatibility.md]: I think it's fine to document without issues. We likely do have issues in pingcap/tidb if you want to search for them though.

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**